src/server/raftkv.rs

```diff
@@ -299,15 +299,17 @@ impl<S: RaftStoreRouter> Engine for RaftKv<S> {
         }
 
         ASYNC_REQUESTS_COUNTER_VEC.write.all.inc();
-        let req_timer = ASYNC_REQUESTS_DURATIONS_VEC.write.start_coarse_timer();
+        let req_timer = Instant::now_coarse();
```

Comment:
Can we use the instant provided by rust-prometheus so that we can use elapsed_secs?